src/registrar/config/settings.py

```diff
@@ -33,11 +33,20 @@
 # Get secrets from Cloud.gov user provided service, if exists
 # If not, get secrets from environment variables
 key_service = AppEnv().get_service(name="getgov-credentials")
+
+
+# Get secrets from Cloud.gov user provided s3 service, if it exists
+s3_key_service = AppEnv().get_service(name="getgov-s3")
+
 if key_service and key_service.credentials:
+    if s3_key_service and s3_key_service.credentials:
+        # Concatenate the credentials from our S3 service into our secret service
+        key_service.credentials.update(s3_key_service.credentials)
     secret = key_service.credentials.get
 else:
     secret = env
```

Comment:
Cloud gov has a built in S3 service. However, this service comes with its own set of AWS credentials. Luckily, we can just grab these directly, so we update  the credentials list